src/app/groups/containers/associated-item/associated-item.component.html

```diff
@@ -1,69 +1,125 @@
-<ng-container *ngIf="state$ | async as state">
-  <h2 class="alg-h2 alg-text-normal title" i18n>
-    Associated {contentType, select, skill {Skill} other {Activity}}
-  </h2>
+@if (isSkill()) { <h2 class="alg-h2 alg-text-normal title" i18n>Associated Skill</h2> }
+@else { <h2 class="alg-h2 alg-text-normal title" i18n>Associated Activity</h2> }
```

Comment:
Mainly the translation is better in this case:
- because of feminine/masculine words in French, often just changing the word is not enough as it requires to change the article and adjective around it (something far). 
- the 2x strings the "select" generates with one which does not have to be translated is quite boring to translate and error prone (as it causes crashes at runtime)
So maybe "select" is better sometimes... (for title it may have been ok), but clearly not every time.